player_openal.go

```diff
@@ -106,6 +106,12 @@ func alFormat(channelNum, bytesPerSample int) C.ALenum {
 
 var numBufs = 2
 
+// SetNumALBuffers sets the underlying number of buffers that oto will use when queueing AL buffers for playback
+// The default is 2, but for higher quality playback 3 or more are recommended
+func SetNumALBuffers(numBuffers int) {
+	numBufs = numBuffers
```

Comment:
Hmm, this exposes implementation details. This is not common in other platforms, right? Isn't it possible to numBufs to 3 by default?